h5p-bildetema/src/components/Bildetema/Bildetema.tsx

```diff
@@ -214,13 +214,13 @@ export const Bildetema: FC<BildetemaProps> = ({
             firstTime={firstTime}
             setFirstTime={setFirstTime}
             handleToggleFavoriteLanguage={handleToggleFavoriteLanguage}
-            hideLanguageSelectors={hidden}
+            hideLanguageSelectors={hideLanguageSelectors}
```

Comment:
This needs the "hideLanguageSelectors" should be for example  languagePathSelectors

Reply:
Yes, to just change the name of the constant?

Reply:
Yes this is for better readability in the code whenever an issue occurs

Reply:
Added